controllers/topics/index.js

```diff
@@ -29,7 +29,7 @@ const UserTopicService = require('../../services/postgres/UserTopicService');
 const getSubscribableTopic = require('./getSubscribeableTopic');
 const {getAnonymUser} = require('../../utils/getAnonymUser');
 const UsersFunction = require('../../databases/functions/users');
-const {insertTopics} = require('../../utils/post');
+const {insertSingleTopic} = require('../../utils/post');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `insertTopics` to `insertSingleTopic` seems fine as long as the new function is designed to handle a single topic insertion. However, it's important to ensure that this change doesn't affect other parts of the codebase where multiple topics might be inserted at once. If there are such cases, consider keeping both functions: `insertTopics` for multiple insertions and `insertSingleTopic` for single insertions.

```diff
- const {insertTopics} = require('../../utils/post');
+ const {insertSingleTopic} = require('../../utils/post');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->